### DISCUSSION TITLE
[react-markdown] Ready-Made Table Component

### DISCUSSION THREAD

**Discussion by StevenXL** [remarkjs/remark#798] [Q&A] [answered]
Hi folks,

I am using `react-markdown`, and I am able to parse my tables fine, in that the HTML produced matches my expectations.

However, I was wondering if there was a ready-made table component through which I can pass the parsed data to get a good-looking table. For example, I use `react-syntax-highlighter` and simply pass in the language and code that was parsed, and the resulting HTML is a nice, syntax-highlighted code block. Is there an equivalent for tables?

**Comment by ChristianMurphy** [comment, marked as answer]
React-markdown doesn't have an opinion on styles or include styles, but there certainly are plenty of options you could choose.

https://github.com/sindresorhus/github-markdown-css can be used without needing to pass a custom component

There are plenty of table components if you want to go that route https://github.com/brillout/awesome-react-components#table

And most style frameworks offer themed tables as well https://reactstrap.github.io/components/tables/, https://material-ui.com/components/tables/, https://v2.grommet.io/table to note a few.
